client/my-sites/domains/controller.jsx

```diff
@@ -102,29 +101,6 @@ const domainSearch = ( context, next ) => {
 	next();
 };
 
-/**
- * If the user enters /domains/add/:domain?domainAndPlanPackage=true,
```

Comment:
Thanks for removing this previously today :)
I didn't find any other occurences of it on our backend.